src/SFA.DAS.AssessorService.Application.Api.Client/Clients/OrganisationsApiClient.cs

```diff
@@ -31,6 +32,16 @@ public async Task<IEnumerable<OrganisationResponse>> GetAll()
             }
         }
 
+        public async Task<OrganisationResponse>  GetOrganisationByName(string name)
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"/api/v1/organisations/{WebUtility.UrlEncode(name)}"))
```

Comment:
First time we've had to URL encode something.

Why not use the inbuild MVC stuff? See comment below :)

Reply:
Yes, we don't need to be manually encoding/decoding.

Reply:
Had a situation when some company names where not  being matched since the default action did not encode special characters properly. But using the manual encode/decode did the job.